.github/actions/multi-approvers/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    "alwaysStrict": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "module": "commonjs",
+    "noImplicitAny": true,
+    "outDir": "./.tsc",
+    "rootDir": "./",
+    "skipLibCheck": true,
+    "strict": true,
+    "target": "es6"
+  },
+  "include": ["src/**/*", "tests/**/*"],
```

Comment:
"exclude": ["node_modules", "**/*.test.ts"]

Reply:
I'd like to understand this more. My goal to to be able to

1. Run the typescript compiler (`tsc`) and compile all TS files in the project (including tests)
2. Be able to view the generated JS for debugging purposes

With your suggestion, test files will not be compiled when running `tsc`. And I don't think we need to exclude node_modules since it's not included in the first place.

Reply:
> I'd like to understand this more. My goal to to be able to
> 
> 1. Run the typescript compiler (`tsc`) and compile all TS files in the project (including tests)

Why? `npm run test` will do all of that for you. You don't need to compile the ts yourself. It's "machine code". Just like you don't go looking into a compiled binary, why do you need to look into the compiled JS

> 2. Be able to view the generated JS for debugging purposes
> 
> With your suggestion, test files will not be compiled when running `tsc`. And I don't think we need to exclude node_modules since it's not included in the first place.

They'll be compiled with `npm run build`, but again, why do you need that?

Reply:
Technically I don't need any of it. I just found it useful when I was developing this. Happy to do it as you wish here since I can adjust setting if I ever want to look at compiles TS again.

Done.